### ISSUE TITLE
Add CSVW parser

### ISSUE THREAD

**Issue by rubensworks** [comunica/comunica#301]
#### Issue type:

- :heavy_plus_sign: Feature request

____
#### Description:

We should be able to parse [CSVW](https://www.w3.org/TR/tabular-data-primer/) documents as described here: https://www.w3.org/TR/csv2rdf/

**Comment by joachimvh** [comment]
Is this not wat RML does?

**Comment by rubensworks** [comment]
Not really, CSVW is a W3C recommendation for representing tabular RDF data with CSV (kind of like JSON-LD, but for CSV). RML is something more generic that converts non-RDF formats to RDF via mapping files, and this is not a W3C recommendation.

**Comment by garpinc** [comment]
This would be a great addition.. an equalizer of sorts.. since common people don't know how to write rdf if they could just add stuff in excel, output in csv and publish to web then we'd have a significant source of new data
